src/content/structured/components/pagination-bar/accessibility.mdx

```diff
@@ -0,0 +1,84 @@
+---
+path: "/components/pagination-bar/accessibility"
+
+date: "2025-07-21"
+
+title: "Pagination bar"
+
+status: "BETA"
+
+subTitle: "Use the pagination bar when large amounts of content are split across multiple pages and various pagination options are required."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/components/pagination-bar"
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/pagination-bar" },
+    { title: "Code", path: "/components/pagination-bar/code" },
+    {
+      title: "Accessibility",
+      path: "/components/pagination-bar/accessibility",
+    },
+  ]
+---
+
+## Easy to use for everyone
+
+The elements within a pagination bar follow a logical tabbing order when moving through them. Buttons are triggered using either the ‘enter’ or ‘space’ key.
+
+The current page is always displayed in the component so that the user knows where in the data set they are.
+
+When the user is at the start or end of the data, the navigation buttons are disabled so that it is clear there is no more data past that point.
+
+Using the items per page selector in the pagination bar allows users to control how much information is available per page. This can help to either simplify complex pages or to increase the information density.
+
+## For Assistive Technology
+
+Each pagination component uses accessible names for buttons that lack meaning in the context of a screen reader. Page buttons that simply have their page number as the visible label are given an accessible label of ‘Go to page X’ so that it is clearer what their function is.
+
+The term ‘page X’ can be overridden in the case that it is not accurate. For example, this can be changed to read ‘sheet X’, ‘set X’, ‘slide X’ instead if it is a better representation.
+
+Similarly, the icon buttons used for next, previous, first and last are also given accessible name so that their action is clearer when announced by a screen reader.
+
+The pagination components use the nav element so that they become navigational landmarks on a page and can therefore be swiftly navigated to by users of assistive technology.
```

Comment:
Sorry - these were already in the pagination guidance but would you mind updating it in this PR? And on the Accessibility page for the pagination component :)
```suggestion
Similarly, the icon buttons used for next, previous, first and last are also given accessible names so that their action is clearer when announced by a screen reader.

The pagination components use the `<nav>` element so that they become navigational landmarks on a page and can therefore be swiftly navigated to by users of assistive technology.
```